wpilibNewCommands/src/test/native/cpp/main.cpp

```diff
@@ -1,16 +1,18 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2015-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2015-2020 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
 #include <hal/HALBase.h>
+#include <networktables/NetworkTableInstance.h>
 
 #include "gtest/gtest.h"
 
 int main(int argc, char** argv) {
   HAL_Initialize(500, 0);
+  nt::NetworkTableInstance::GetDefault().StartLocal();
```

Comment:
It seems like this shouldn't be global, but should rather be part of a test fixture setup and teardown.